docs/delayed_embed.md

```diff
@@ -0,0 +1,116 @@
+# Delayed embed
+
+You may decide to combine pgai with [Timescaledb Background Actions][tsdb_actions].
+and you can also use the `pgai` extension to run background actions.
```

Comment:
Should we also provide instructions for using pg_cron?

Reply:
That's a good point, I started with timescale jobs but it maybe also a good tutorial we can build and make a lighter dependency.

Reply:
I'd go with it in another PR to just simplify the merge for now and then we introduce the new example.